VWOSdk/VWO.cs

```diff
@@ -136,24 +162,40 @@ public static Settings GetAndUpdateSettingsFile(long accountId, string sdkKey)
         /// IVWOClient instance to call Activate, GetVariation and Track apis for given user and goal.
         /// </returns>
         public static IVWOClient Launch(Settings settingFile, bool isDevelopmentMode = false, IUserStorageService userStorageService = null,
-           BatchEventData batchData = null, string goalTypeToTrack = Constants.GoalTypes.ALL,
+           BatchEventData batchData = null, string goalTypeToTrack = null,

```

Comment:
why is it changes?

Reply:
Me and Sanyam discussed on that and we had to do for usage stats .